consultation_analyser/consultations/models.py

```diff
@@ -1,7 +1,43 @@
-# Create your models here.
 from django.db import models
 
 
+class Consultation(models.Model):
+    # May need something better long-term
+    class Organisation(models.TextChoices):
+        DFE = "dept-for-education", "Department for Education"
+        CO = "cabinet-office", "Cabinet Office"
+
+    identifier = models.CharField(max_length=256, blank=False)
+    name = models.CharField(max_length=256, blank=False)
+    organisation_identifier = models.CharField(max_length=256, choices=Organisation.choices, blank=True)
+
+
 class Question(models.Model):
     text = models.CharField(max_length=None)  # no idea what's a sensible value for max_length
     slug = models.CharField(max_length=256)
+    is_closed = models.BooleanField(default=False)
```

Comment:
`is_closed` and `is_open` feel like they're mutually exclusive — isn't the key question whether there's a closed part? Would the presence of `closed_response_options` be enough to give us that information?

Reply:
Yup, the presence of `closed_response_options` is enough - so we can remove `is_closed`. 

Do you think `is_open` could be `has_free_text_response` or to make it more explicit?

Reply:
Yes to more explicit!

Reply:
Field names now updated to:
```
    has_free_text = models.BooleanField(default=False)
    multiple_choice_options = models.JSONField(default=list)
```